playbooks/tests/tasks/lbaas.yml

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
I'm of mixed opinion. Rally is a great tool for a shotgun approach of test all the things, but often the part that may be malfunctioning is hard to pinpoint as the whole things is somewhat binary pass/fail. Individual stressors of a given subsystem may more quickly uncover health issues with that particular subsystem.

I don't think we should go beyond the very basics in our hand written tests, relying on Rally and Tempest for the more in-depth coverage. The simple tests give us a fail early possibility.